app/views/notification_mailer/ticket_assigned.text.erb

```diff
@@ -5,7 +5,7 @@
 <%= t('notify_ticket_assigned', default: 'You have recently been assigned to the following topic(s):') %>
 
 <% @bulk_topics.each do |topic| %>
-  <%= topic.name %>
+  <%= topic.name + '  '%><%= link_to t('view_on_site', default: "View on Site"), topic %>
```

Comment:
Same goes here with `' '`.
Also the `link_to` helper returns the html anchor tag (`<a href=.....>`)
We just want to insert the `admin_topic_url(topic, host: AppSettings['settings.site_url'])`

Also, the `Reply directly to this email or view this online` line should probably only be shown if the assignment is for 1 topic. (line 11 of this file)